ca_gazebo/models/apriltag2/model.config

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<model>
+    <name>demo</name>
```

Comment:
Use a proper name and fill the other information accordingly.

Reply:
Adressed